### DISCUSSION TITLE
Colours from DB

### DISCUSSION THREAD

**Discussion by Zathire** [Budibase/budibase#2514] [Ideas]
Any chance in a future release we might see the ability to set colours based from a DB table column?

I'm building an app that has colour coded categories and would like to style borders based on the categories colour in the database.

**Comment by joebudi** [comment]
Hey! There is a possibility but not in the next couple of months.

**Reply by Zathire** [reply]
Appreciate the quick response thanks!

No worries, I'll work with it until them :-)

**Reply by joebudi** [reply]
If you have any other questions, please feel free to ask :-).
